lib/manager/gradle-wrapper/utils.ts

```diff
@@ -72,20 +74,29 @@ export function getJavaVersioning(): string {
   return npmVersioning;
 }
 
-// https://regex101.com/r/1GaQ2X/1
+// https://regex101.com/r/IcOs7P/1
 const DISTRIBUTION_URL_REGEX = regEx(
-  '^(?:distributionUrl\\s*=\\s*)\\S*-(?<version>\\d+\\.\\d+(?:\\.\\d+)?(?:-\\w+)*)-(?<type>bin|all)\\.zip\\s*$'
+  '^(?:distributionUrl\\s*=\\s*)(?<url>\\S*-(?<version>\\d+\\.\\d+(?:\\.\\d+)?(?:-\\w+)*)-(?<type>bin|all)\\.zip)\\s*$'
 );
 
-export function extractGradleVersion(fileContent: string): string | null {
+export function extractGradleVersion(
+  fileContent: string
+): GradleVersionExtract | null {
   const lines = fileContent?.split('\n') ?? [];
 
   for (const line of lines) {
     const distributionUrlMatch = DISTRIBUTION_URL_REGEX.exec(line);
+
     if (distributionUrlMatch) {
-      return distributionUrlMatch.groups.version;
+      return {
+        url: distributionUrlMatch.groups.url,
+        version: distributionUrlMatch.groups.version,
+      };
     }
   }
+  logger.info(
```

Comment:
@rarkins should we issue a warning aor debug here? 🤔 Normally we just silently ignore non matching stuff

Reply:
I've added the log statement, to help useres understand, why there are no updates to their Gradle Wrapper, even though their project contains one.
As this is basically an error based on a user configuration, namely the way the Gradle Wrapper is set up, I've added a logger.info statement in accordance with the [documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/design-decisions.md#logging-and-error-levels).

This was just a pitfall I've encountered while setting up Renovate, thus I thought it would be helpful to others to get at least a log entry.

Reply:
I think we should lower this to debug message, as we do it in other managers.

Reply:
Lowered the log level to debug.

Thanks for the review. 👍 
I'll mark this as resolved, maybe you can merge then?